openmdao/core/system.py

```diff
@@ -191,14 +210,20 @@ def _setup_paths(self, parent_path):
         parent_path : str
             The pathname of the parent `System`, which is to be prepended to the
             name of this child `System`.
+
+        probdata : `_ProbData`
+            Problem level data container.
         """
-        self._fd_params = None
+        self._reset()
 
         if parent_path:
             self.pathname = '.'.join((parent_path, self.name))
         else:
             self.pathname = self.name
 
+        self._sysdata = _SysData(self.pathname)
+        self._probdata = probdata
+
```

Comment:
yeah, I just added it there because setup_paths is the first call in setup and I didn't want to add another setup related api function.  I'm open to suggestions for a rename, maybe setup_init() or something like that?